app/src/sharedTest/java/org/oppia/android/app/recyclerview/RecyclerViewMatcher.kt

```diff
@@ -102,4 +106,30 @@ class RecyclerViewMatcher {
       assertThat("RecyclerViewGrid span count", spanCount, equalTo(count))
     }
   }
+
+  private class GridLayoutManagerColumnCountAssertion(expectedColumnCount: Int) : ViewAssertion {
+    private var expectedColumnCount: Int = 0
+
+    init {
+      this.expectedColumnCount = expectedColumnCount
+    }
+
+    override fun check(view: View, noViewFoundException: NoMatchingViewException?) {
+      if (noViewFoundException != null) {
+        throw noViewFoundException
+      }
+      val recyclerView = view as RecyclerView
+      if (recyclerView.layoutManager is GridLayoutManager) {
+        val gridLayoutManager = recyclerView.layoutManager as GridLayoutManager
+        val spanCount = gridLayoutManager.spanCount
+        if (spanCount != expectedColumnCount) {
+          val errorMessage =
+            ("expected column count " + expectedColumnCount + " but was " + spanCount)
```

Comment:
Nit:
`("expected column count $expectedColumnCount but was $spanCount")`

Reply:
@anandwana001 feel free to resolve on your own.

Reply:
Done